scripts/makefilegen.py

```diff
@@ -53,6 +53,7 @@
 argparser.add_argument(      '--contentid', help='Specify 64-bit contentid for PCIe designs')
 argparser.add_argument('-I', '--cinclude', help='Specify C++ include directories', default=[], action='append')
 argparser.add_argument('-V', '--verilog', default=[], help='Additional verilog sources', action='append')
+argparser.add_argument('-SV', '--systemverilog', default=[], help='Additional systemverilog sources', action='append')
```

Comment:
now that I understand argparser better, I suggest adding default=[] here and removing the conditional assignment below. But I should do that for --verilog, etc, so you don't have to change this pull request.